java/src/jmri/jmrit/dispatcher/DispatcherFrame.java

```diff
@@ -1451,7 +1456,7 @@ public ActiveTrain createActiveTrain(String transitID, String trainID, int tSour
             return null;
         }
         if (_HasOccupancyDetection && (!(startBlock.getState() == Block.OCCUPIED))) {
-            if (showErrorMessages) {
+            if (showErrorMessages && !this.dispatcherSystemSchedulingInOperation) {
```

Comment:
Static analysis CI fails on this line.

Replace `this.dispatcherSystemSchedulingInOperation` with `DispatcherFrame.dispatcherSystemSchedulingInOperation`.

(Replace `this` with `DispatcherFrame`)